app/controllers/admin/terms_of_service/drafts_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class Admin::TermsOfService::DraftsController < Admin::BaseController
+  before_action :set_terms_of_service
+
+  def show
```

Comment:
This view is basically a form ... should it be `edit` action?

Reply:
I don't think the URL needs to be `/draft/edit`, which is why I'm going with `show`. There is no practical difference and there are other controllers that use `show` for forms.